src/views/v2/tickets/item/extra/price.php

```diff
@@ -56,7 +56,7 @@
 ?>
 <div <?php tribe_classes( $classes ); ?>>
 	<?php if ( ! empty( $ticket->on_sale ) ) : ?>
-		<span class="tribe-common-b2 tribe-tickets__tickets-original-price">
+		<span class="tribe-common-b3 tribe-tickets__tickets-original-price">
```

Comment:
We'll need to set the `@since TBD changes description.` for this template. And the `@version TBD`.